docs/worldgen/biomemodifier.md

```diff
@@ -695,6 +697,24 @@ This will then result in the following JSON being created:
 }
 ```
 
+## Targeting Biomes That May Not Exist
+
+There may be times when a biome modifier needs to target a biome that is not always present in the game. If a biome modifier targets the unregistered biome directly, it will crash on world loading. The way to work around this is to create a biome tag and add the target biome as an optional tag entry by setting required to false for the entry. An example is below:
+
+```json5
+{
+  "replace": false,
+  "values": [
+    {
+      "id": "minecraft:pale_garden",
+      "required": false
+    }
+  ]
+}
+```
```

Comment:
I would also provide a method for how to datagen this using a `TagsProvider`

Reply:
I don’t know how to datagen. Would you be able to add a commit to this PR of what that should look like?

Reply:
```java
// In a TagsProvider<Biome> subclass
// Assume we have some example TagKey<Biome> OPTIONAL_BIOMES_TAG
@Override
protected void addTags(HolderLookup.Provider registries) {
    this.tag(OPTIONAL_BIOMES_TAG)
        // Must be a ResourceLocation representing the registry object
        .addOptional(WinterDropBiomes.PALE_GARDEN.location());
}
```

Should be something like that.